Stack/Opc.Ua.Core/Stack/Tcp/TcpServerChannel.cs

```diff
@@ -19,43 +19,7 @@
 using System.Threading.Tasks;
 using System.Net.Sockets;
 
-namespace Opc.Ua.Bindings
-{
-    /// <summary>
-    /// Interface between listener and UA TCP channel
-    /// </summary>
-    public interface ITcpServerChannelListener
-    {
-        /// <summary>
-        /// The endpoint url of the listener
-        /// </summary>
-        Uri EndpointUrl { get; }
-
-        /// <summary>
-        /// Binds a new socket to an existing channel.
-        /// </summary>
-        /// <param name="socket"></param>
-        /// <param name="requestId"></param>
-        /// <param name="sequenceNumber"></param>
-        /// <param name="channelId"></param>
-        /// <param name="clientCertificate"></param>
-        /// <param name="token"></param>
-        /// <param name="request"></param>
-        /// <returns></returns>
-        bool ReconnectToExistingChannel(
-            IMessageSocket socket,
-            uint requestId,
-            uint sequenceNumber,
-            uint channelId,
-            X509Certificate2 clientCertificate,
-            ChannelToken token,
-            OpenSecureChannelRequest request);
-
-        /// <summary>
-        /// Called when a channel closes.
-        /// </summary>
-        void ChannelClosed(uint channelId);
-    }
```

Comment:
ITcpChannelListener file is missing now..

Reply:
Oops.  Added it, some git add malfunction :-)